packages/flutter/lib/src/material/menu_anchor.dart

```diff
@@ -696,6 +696,73 @@ class MenuController {
   }
 }
 
+/// A [MenuController] that allows listening to its [isOpen] state.
```

Comment:
Feel free to suggest better docs.

Should we also have a `See also` that links to the `MenuController` / `MenuAnchor` ?